gensim/models/doc2vec_inner.pyx

```diff
@@ -94,7 +96,8 @@ cdef unsigned long long fast_document_dbow_neg(
         if learn_hidden:
             our_saxpy(&size, &g, &context_vectors[row1], &ONE, &syn1neg[row2], &ONE)
     if learn_context:
-        our_saxpy(&size, &context_locks[context_index], work, &ONE, &context_vectors[row1], &ONE)
+        our_saxpy(&size, &contexts_lockf[context_index % contexts_lockf_len],
```

Comment:
This 'aligned with opening delimiter' indentation style happens automatically in emacs `python-mode`, & is compatible with PEP8 & other major influential Python style guides (like the Google Python Style Guide). It's also already commonly seen in the `gensim` codebase. It's not caught by our automated style-checking. So if in fact your preference is to tighten new work to a standard beyond PEP8/Google for `gensim`, I'd suggest that (1) that creates an extra barrier to contributions from people writing to PEP8/etc; (2) if still a requirement, it happen as a separate, later cleanup step, so as not to distract from other more substantive changes.

Reply:
I hope not! We've been quite relentless about hanging indent, so if you still see it in the code base, consider it a bug.

We can do it in a separate step though I'll likely simply fix them when I submit my PR against this PR, it's trivial.

Reply:
Do you really want to be tighter than PEP8 & Google code style & emacs `python-mode` defaults? Do you have an automated tool flagging these, or just using your eyes?

Reply:
Just using our eyes + I believe `black` can do this automatically. IIRC we had a ticket for enabling black but it's low priority – these are so easy to spot & correct in PRs that it may not warrant bringing in yet another machinery.

Reply:
But why do you want to be tighter than PEP8 & Google here? It will be a constant drag on the productivity of contributors coming from the larger community.